coordinator/src/models/fhir/fhir-resources.ts

```diff
@@ -78,10 +82,11 @@ export interface IdentifierReference<T extends Resource> {
 export interface Dosage {
   text: string
   patientInstruction?: string
+  additionalInstruction?: Array<CodeableConcept>
 }
 
 export interface Performer extends IdentifierReference<Organization> {
-  extension: Array<ReferenceExtension<PractitionerRole>>
+  extension?: Array<ReferenceExtension<PractitionerRole>>
```

Comment:
Does it make sense that the only field in this interface is nullable?

Reply:
had to make this change since i was building medicationRequest objects that didnt have a performer block in them (due to difference between prescription-order & prescription-order-update MRs). i guess it makes sense in a way since a performer block is just a particular IdentifierReference that may have an extension block too.

Reply:
Think I understand. It might be clearer if this was renames `IdentifierReferenceWithExtension` (or something similar) and the extension wasn't nullable. Then `Performer` could be `IdentifierReferenceWithExtension || Identifierreference`. Not sure, but that feels neater to me?

Reply:
Maybe I'm just overthinking it

Reply:
and it might confuse things further down the line, idk